pkg/kubectl/cmd/config/view.go

```diff
@@ -106,6 +109,11 @@ func (o ViewOptions) Run(out io.Writer, printer kubectl.ResourcePrinter) error {
 		}
 	}
 
+	if o.Short {
+		fmt.Printf("short\n")
```

Comment:
why the extra output? won't this mess up anything expecting structured output, or anything using templated output?